src/SFA-DAS-HMRC-API-Stub/Data/Repositories/Cosmos/FractionCalDateCosmosRepository.cs

```diff
@@ -0,0 +1,40 @@
+﻿using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.HMRC.API.Stub.Domain;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.HMRC.API.Stub.Data.Repositories
+{
+    public class FractionCalDateCosmosRepository : BaseCosmosRepository, IFractionCalDateRepository
+    {
+        private readonly ILogger<FractionCalDateCosmosRepository> _logger;
+
+        public FractionCalDateCosmosRepository(DocumentClient client, ILogger<FractionCalDateCosmosRepository> logger, Uri collectionUri)
+            : base(client, collectionUri)
+        {
+            _logger = logger ?? throw new ArgumentException("logger cannot be null");
+        }
+
+        public async Task<FractionCalculationDate> GetByLastCalDate(
+            DateTime? lastCalculationDate)

```

Comment:
There is no data needed to be passed into the repository so no method param is needed for lastCalculationDate